configtx/msp.go

```diff
@@ -74,6 +75,393 @@ type MSP struct {
+}
+
+// CreateMSPCRL creates a CRL that revokes the provided certificates
+// for the specified organization's msp signed by the provided SigningIdentity.
+func (m *MSP) CreateMSPCRL(signingIdentity *SigningIdentity, certs ...*x509.Certificate) (*pkix.CertificateList, error) {
+	return m.newMSPCRL(signingIdentity, certs...)
+}
+
+// Configuration returns the MSP value for an orderer organization in the
+// updated config.
+func (m *OrganizationMSP) Configuration() (MSP, error) {
+	return getMSPConfig(m.configGroup)
+}
+
+// AddAdminCert adds an administator identity to the organization MSP.
+func (m *OrganizationMSP) AddAdminCert(cert *x509.Certificate) error {
+	msp, err := getMSPConfig(m.configGroup)
+	if err != nil {
+		return err
+	}
+
+	for _, c := range msp.Admins {
+		if c.Equal(cert) {
+			return nil
+		}
+	}
+
+	msp.Admins = append(msp.Admins, cert)
+
+	return msp.setConfig(m.configGroup)
+}
+
+// RemoveAdminCert removes an administator identity from the organization MSP.
+func (m *OrganizationMSP) RemoveAdminCert(cert *x509.Certificate) error {
+	msp, err := getMSPConfig(m.configGroup)
+	if err != nil {
+		return err
+	}
+
+	certs := msp.Admins[:]
+	for i, c := range msp.Admins {
+		if c.Equal(cert) {
+			certs = append(certs[:i], certs[i+1:]...)
+			break
+		}
+	}
+
+	msp.Admins = certs
+
+	return msp.setConfig(m.configGroup)
+}
+
+// AddRootCert adds a root certificate trusted by the organization MSP.
+func (m *OrganizationMSP) AddRootCert(cert *x509.Certificate) error {
+	msp, err := getMSPConfig(m.configGroup)
+	if err != nil {
+		return err
+	}
+
+	for _, c := range msp.RootCerts {
+		if c.Equal(cert) {
+			return nil
+		}
+	}
+
+	msp.RootCerts = append(msp.RootCerts, cert)
+
+	err = msp.validateCACerts()
+	if err != nil {
+		return err
+	}
+
+	return msp.setConfig(m.configGroup)
+}
+
+// RemoveRootCert removes a trusted root certificate from the organization MSP.
+func (m *OrganizationMSP) RemoveRootCert(cert *x509.Certificate) error {
+	msp, err := getMSPConfig(m.configGroup)
+	if err != nil {
+		return err
+	}
+
+	certs := msp.RootCerts[:]
+	for i, c := range msp.RootCerts {
+		if c.Equal(cert) {
+			certs = append(certs[:i], certs[i+1:]...)
+			break
+		}
+	}
+
+	msp.RootCerts = certs
+
+	err = msp.validateCACerts()
+	if err != nil {
+		return err
+	}
+
+	return msp.setConfig(m.configGroup)
+}
+
+// AddIntermediateCert adds an intermediate certificate trusted by the organization MSP.
+func (m *OrganizationMSP) AddIntermediateCert(cert *x509.Certificate) error {
+	msp, err := getMSPConfig(m.configGroup)
+	if err != nil {
+		return err
+	}
+
+	for _, c := range msp.IntermediateCerts {
+		if c.Equal(cert) {
+			return nil
+		}
+	}
+
+	msp.IntermediateCerts = append(msp.IntermediateCerts, cert)
+
+	err = msp.validateCACerts()
+	if err != nil {
+		return err
+	}
+
+	return msp.setConfig(m.configGroup)
+}
+
+// RemoveIntermediateCert removes a trusted intermediate certificate trusted by the organization MSP.
```

Comment:
lot of `trusted` here :) maybe remove one of them.
`RemoveIntermediateCert removes an intermediate certificate trusted by the organization MSP.` or `RemoveIntermediateCert removes a trusted intermediate certificate from the organization MSP.` Your choice, same comment applies to the other comments that use `trusted` redundantly.

Reply:
🛠️ looks like this was the only place with multiple uses of trusted